src/components/Fund/FundDetails/FundDetailsContainer.js

```diff
@@ -0,0 +1,280 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  MenuSection,
+  Pane,
+  PaneMenu,
+  Row,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+import { LoadingPane } from '@folio/stripes-acq-components';
+
+import {
+  DetailsEditAction,
+  DetailsRemoveAction,
+} from '../../../common/DetailsActions';
+import {
+  acqUnitsResource,
+  fundResource,
+  fundsResource,
+  fundTypesResource,
+  groupsResource,
+  ledgersResource,
+} from '../../../common/resources';
+import { SECTIONS_FUND } from '../constants';
+import FundDetails from './FundDetails';
+
+const FundDetailsContainer = ({
+  match: { params },
+  mutator,
+  onClose,
+  onEdit,
+  resources,
+}) => {
+  useEffect(() => {
+    mutator.fund.reset();
+    mutator.fundType.reset();
+    mutator.ledger.reset();
+    mutator.allocatedFrom.reset();
+    mutator.allocatedTo.reset();
+    mutator.group.reset();
+    mutator.acqUnits.reset();
+
+    mutator.fund.GET().then(response => {
+      const {
+        ledgerId,
+        fundTypeId,
+        allocatedFromIds,
+        allocatedToIds,
+        acqUnitIds,
+      } = response;
+
+      if (fundTypeId) {
+        mutator.fundType.GET({
+          params: {
+            query: `id==${fundTypeId}`,
+          },
+        });
+      }
+
+      if (ledgerId) {
+        mutator.ledger.GET({
+          params: {
+            query: `id==${ledgerId}`,
+          },
+        });
+      }
+
+      if (allocatedFromIds.length) {
+        mutator.allocatedFrom.GET({
+          params: {
+            query: allocatedFromIds.map(fundId => `id==${fundId}`).join(' or '),
+          },
+        });
+      }
+
+      if (allocatedToIds.length) {
+        mutator.allocatedTo.GET({
+          params: {
+            query: allocatedToIds.map(fundId => `id==${fundId}`).join(' or '),
+          },
+        });
+      }
+
+      if (acqUnitIds.length) {
+        mutator.acqUnits.GET({
+          params: {
+            query: acqUnitIds.map(id => `id==${id}`).join(' or '),
+          },
+        });
+      }
+    });
+
+    mutator.group.GET({
+      params: {
+        query: 'fundId==:{id}',
+      },
+    });
+  }, [params.id]);
+
+  const [sections, setSections] = useState({});
+
+  const expandAll = useCallback(
```

Comment:
actually, don't we have common code for this? AFAIK, it's a copy-paste, it would be great to use something common if it exists